src/js/media-container.js

```diff
@@ -17,6 +17,8 @@ import { nouns } from './labels/labels.js';
 import { containsComposedNode } from './utils/element-utils.js';
 // Guarantee that `<media-gesture-receiver/>` is available for use in the template
 import './media-gesture-receiver.js';
+import MediaChromeRange from './media-chrome-range.js';
```

Comment:
This makes me cringe a little. Partly for the added dependency complexity, but partly because it means we've made media chrome's button and range more special somehow in this ecosystem. I'm not gonna call this a blocker but want to encourage us to consider an approach that removes this dependency, maybe as a fast follow.

Reply:
Yeah, this is mostly for to get it working. I think the proper solution is what we discussed in https://github.com/muxinc/media-chrome/discussions/257, where the elements can expose a property like `keysUsed` and then below, we can do
```js
// skip handler is already used by the target
if (evt.target.keysUsed ?? []).includes(evt.key)) {
  return;
}
```
And we won't need to import these directly.

The alternative is that each component that handles keyup will call `stopPropagation` properly.

Originally, I was thinking we can punt on this, but this affects a component that's currently in Mux Player, so, we probably should come to a decision sooner rather than earlier. I lean towards a properly like `keysUsed` (or another name).

Reply:
Ok cool, makes sense. 👍 

Without understanding all the complexity, I still feel better about something where control-specific complexity is kept in the controls, or the super classes of controls, rather than making media container/controller smarter here so controls don't have to be. Probably worth a call to discuss.

Reply:
Added a new comment so that this doesn't get lost here https://github.com/muxinc/media-chrome/pull/263#discussion_r919493069